build/settings.js

```diff
@@ -38,7 +38,8 @@ settings = {
   remoteUrl: 'https://api.resin.io',
   dashboardUrl: 'https://dashboard.resin.io',
   dataDirectory: path.join(userHome, '.resin'),
-  imageCacheTime: 1 * 1000 * 60 * 60 * 24 * 7
+  imageCacheTime: 1 * 1000 * 60 * 60 * 24 * 7,
+  tokenValidityTime: 1 * 1000 * 60 * 60
```

Comment:
Yeah, as it stands I would very much expect this to mean that any token older than this is invalid/expired, which it's not.  In fact the refreshes of the token can happen in parallel to the actual user request (since the current token still has to be valid - or the refresh would fail anyway)